resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -205,130 +216,131 @@
-      type: list:basic:string
-      required: false
-    - name: down_primers_seq_rev
-      label: 3 prime reverse reads adapter sequence
-      type: list:basic:string
-      required: false
-    - name: polya_tail
-      label: Poly-A tail
-      type: basic:integer
-      required: false
-      description: >
-        Length of poly-A tail, example - AAAN -> 3, AAAAAN -> 5
-    - name: leading
-      label: Leading quality
-      type: basic:integer
-      required: false
-      description: >
-        Remove low quality bases from the beginning. Specifies the minimum quality required to keep a base.
-    - name: trailing
-      label: Trailing quality
-      type: basic:integer
-      required: false
-      description: >
-        Remove low quality bases from the end. Specifies the minimum quality required to keep a base.
-    - name: crop_fwd
-      label: Crop (forward reads)
-      type: basic:integer
-      required: false
-      description: >
-       Cut the specified number of bases from the end of the forward reads
-    - name: crop_rev
-      label: Crop (reverse reads)
-      type: basic:integer
-      required: false
-      description: >
-       Cut the specified number of bases from the end of the reverse reads
-    - name: headcrop_fwd
-      label: Headcrop (forward reads)
-      type: basic:integer
-      required: false
-      description: >
-       Cut the specified number of bases from the start of the forward reads
-    - name: headcrop_rev
-      label: Headcrop (reverse reads)
-      type: basic:integer
-      required: false
-      description: >
-       Cut the specified number of bases from the start of the reverse reads
-    - name: min_overlap_fwd
-      label: Minimal overlap (forward reads)
-      type: basic:integer
-      default: 3
-      description: >
-       Minimum overlap for an adapter match in forward reads
-    - name: min_overlap_rev
-      label: Minimal overlap (reverse reads)
-      type: basic:integer
-      default: 3
-      description: >
-       Minimum overlap for an adapter match in reverse reads
-    - name: minlen
-      label: Min length
-      type: basic:integer
-      required: false
-      description: >
-        Drop the read if it is below a specified length
-    - name: match_read_wildcards
-      label: Match read wildcards
-      type: basic:boolean
-      required: false
-      default: false
-      description: >
-        Interpret IUPAC wildcards in reads.
-    - name: times
-      label: Times
-      type: basic:integer
-      required: false
-      default: 1
-      description: >
-        Remove up to COUNT adapters from each read.
-    - name: error_rate
-      label: Error rate
-      type: basic:decimal
-      required: false
-      default: 0.1
-      description: >
-        Maximum allowed error rate (no. of errors divided by the length of the matching region).
+    - name: adapters
+      label: Adapters
+      group:
+        - name: up_primers_file_fwd
```

Comment:
This is a matter of debate, but IMO, it would be better to rename this to `5_prime_mate_1`. Terms `up`, `down`, `forward` and `reverse` can be confusing.

Reply:
I agree

Reply:
I have applied the suggestion only in `label` filed, because we don't support numbers in `name`.

Reply:
You could use `mate1_5prime` or something similar... It would be more in sync with the label. But it is also fine as it is now.